triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -709,14 +723,14 @@ private[lf] class Runner private (
     val graph = GraphDSL.createGraph(Sink.last[SValue]) { implicit gb => saveLastState =>
       import GraphDSL.Implicits._
 
-      val msgIn = gb add Flow[TriggerMsg].wireTap(logReceivedMsg _)
+      val msgIn = gb add Flow[TriggerMsg]
```

Comment:
In a future PR, `Ctx` based logging on the flow will render this logging useless - removing it now for simplicity